client/js/components/procedure/admin/InstitutionTagManagement/InstitutionTagManagement.vue

```diff
@@ -26,7 +26,7 @@
       :is-active="activeTabId === 'tagList'"
       :label="Translator.trans('tag.administrate')">
       <slot>
-        <tag-list @tagIsRemoved="institutionListReset" />
+        <tag-list @tag-is-removed="institutionListReset" />
```

Comment:
I think the emit is still `tagIsRemoved`. Does vue recoginze this with `@tag-is-removed` (same pattern as props?)?

Reply:
It's actually the recommended way of doing it in vue 3 (see https://vuejs.org/guide/components/events + https://eslint.vuejs.org/rules/v-on-event-hyphenation.html), but I'm not sure we want to do it like that. I like it better when you can search for the event name and find all usages, so I disabled the eslint rule for now. This change was accidental, reverted in 92dba4713b5255f4495849a5f5e4d3d8b7d5bdea